### ISSUE TITLE
Make it optional to save errors to a temp file

### ISSUE THREAD

**Issue by vrikselzi** [clj-clapps/clj-clapps#4]
Some programs require showing the error messages to the user. Currently when errors occur, they are saved to a temp file.
Could saving the errors to a file be optional? Maybe by adding a global command option like `:show-cmd-errors?` which would print out the errors if `true` or save them to a file if `false`.

**Comment by rinconjc** [comment]
Yeah that makes sense.  Maybe the existing verbose option (-v) could be used for it.

**Comment by vrikselzi** [comment]
Thanks! I tried the fix and error messages show when they occur. Although I could not get the full stack trace when using the verbose option `-v`. Using the option in a command returns the message:
```
Invalid command options and/or arguments:
Unknown option: "-v"
```
Tested on a Mac.

**Comment by rinconjc** [comment]
It seems the verbosity option was missing for sub-commands and single commands. Please try again with the latest commit.
